src/1_utils.js

```diff
@@ -661,3 +697,13 @@ utils.openGraphDataAsObject = function() {
 	};
 };
 
+utils.appendBfpToLinkUrl = function(url, bfp) {
+	if (typeof url !== 'string') {
+		return null;
+	}
+	if (!bfp) {
+		return url;
+	}
+
+	return url + '?_t=' + bfp;
```

Comment:
can we check for the existence of a `?` before appending the bfp in case v1/url returns a query param? It doesn't happen right now but something similar happened on the mobile sdks before and it is still biting us.